.github/workflows/unit-tests.yaml

```diff
@@ -29,12 +32,15 @@ jobs:
       run: |
         # Prevent fatal error "detected dubious ownership in repository" from recent git.
         git config --global --add safe.directory "$(pwd)"
+        su tester -c "git config --global --add safe.directory \"$(pwd)\""
 
     - name: Cache Go dependencies
       uses: ./.github/actions/cache-go-dependencies
 
     - name: Go Unit Tests
-      run: ${{ matrix.gotags }} make go-unit-tests
+      run: |
+        chmod a+w . roxctl/connectivity-map/testdata
```

Comment:
This is actually a bats test for roxctl, not the go unit test - mentioning only because of the name of that step.

Reply:
Is it?

Reply:
Or maybe it is used in both? It can stay like this